connections/management/commands/send_oikotie_xml_file.py

```diff
@@ -16,40 +16,62 @@
 
 
 class Command(BaseCommand):
-    help = "Generate apartments and housing companies XML files to be shown in Oikotie \
-and send them via FTP"
+    help = (
+        "Generate apartments and housing companies XML files to be shown in Oikotie "
+        "and send them via FTP"
+    )
 
     def add_arguments(self, parser):
         parser.add_argument(
             "--only_create_files",
             action="store_true",
             help="Only create XML files without sending them via FTP",
         )
+        parser.add_argument(
+            "--send_only_type",
+            type=int,
+            choices=[1, 2],
+            help="Send either housing company file (1) or apartment file (2)",
+        )
 
     def handle(self, *args, **options):
         path = settings.APARTMENT_DATA_TRANSFER_PATH
         apartments, housing_companies = fetch_apartments_for_sale()
-        apartment_file = create_xml_apartment_file(apartments)
-        housing_file = create_xml_housing_company_file(housing_companies)
-
-        if not options["only_create_files"] and apartment_file and housing_file:
-            for f in [apartment_file, housing_file]:
-                try:
-                    send_items(path, f)
-                    _logger.info(
-                        f"Succefully sent XML file {path}/{f} to Oikotie FTP server"
-                    )
-                except Exception as e:
-                    _logger.error(
-                        f"File {path}/{f} sending via FTP to Oikotie failed:",
-                        str(e),
-                    )
-                    raise e
+        sending_apartments = False
+        oikotie_files = []
+
+        if not options["send_only_type"] or options["send_only_type"] == 1:
+            oikotie_files.append(create_xml_housing_company_file(housing_companies))
+
+        if not options["send_only_type"] or options["send_only_type"] == 2:
+            sending_apartments = True
+            oikotie_files.append(create_xml_apartment_file(apartments))
+
+        if options["only_create_files"]:
+            _logger.info("Not sending XML files to Oikotie")
+            return
+
+        for f in oikotie_files:
```

Comment:
Use full name in variables instead of the single character. It is easier to read in the code and describes what the variable is e.g. file_name.